packages/terra-hyperlink/src/Hyperlink.scss

```diff
@@ -0,0 +1,97 @@
+@import '~terra-mixins';
+
+:local {
+  .hyperlink {
+    background-color: transparent; // Remove the gray background on active links in IE 10
+    color: var(--terra-link-color, #0059a8);
+    display: inline-block; // prevents variant icon from breaking onto its own line
+    outline: none;
+    text-decoration: var(--terra-link-underline, none);
+
+    &:visited,
+    &.is-visited {
+      color: var(--terra-link-visted-color, #0059a8);
+    }
+
+    &:hover,
+    &.is-hovered {
+      color: var(--terra-link-hover-color, #001f67);
+      text-decoration: var(--terra-link-hover-underline, underline);
+    }
+
+    &.is-focused {
+      background-color: var(--terra-link-focus-background-color, transparent);
+      border-radius: var(--terra-link-focus-border-radius, 0.2142rem);
+      box-shadow: var(--terra-link-focus-box-shadow, 0 0 0 1px #4cb2e9, 0 0 0 3px rgba(76, 178, 233, 0.25));
+      color: var(--terra-link-focus-color, #001f67);
+      text-decoration: var(--terra-link-focus-underline, underline);
+    }
+
+    &.is-active {
+      color: var(--terra-link-active-color, #001f67);
+    }
+
+    &.is-disabled {
+      color: var(--terra-link-disabled-color, rgba(28, 31, 33, 0.2));
+      text-decoration: none;
+    }
+  }
+
+  .is-underline-hidden {
+    text-decoration: none;
+
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  // Icons
+  .external,
+  .image,
+  .video,
+  .audio,
+  .document {
+    &::after {
+      background-repeat: no-repeat;
+      background-size: cover;
+      content: '';
+      display: inline-block;
+      height: 1em;
+      margin-left: 0.5em;
+      position: relative;
+      top: 0.15em;
+      width: 1em;
+    }
+  }
+
+  .external {
+    &::after {
+      background-image: var(--terra-link-external-icon, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#0059a8" d="M39 25h6v23H0V3h23v6H6v33h33V25zm-21.4 9.6l24-24L48 17V0H31l6.4 6.4-24 24 4.2 4.2z"></path></svg>'));
```

Comment:
We may want to check with UX about these icons. I looked up the Zeplin designs and it looks like the icons are supposed to change color with the various states. This is easiest to see with the consumer theme design, but is visible in the base implementation for the `isDisabled` state.

You can see what I had to do for Paginator to accomodate that: https://github.com/cerner/terra-core/blob/master/packages/terra-paginator/src/Paginator.module.scss#L89..L113

Reply:
Talked with @neilpfeiffer about this. Each variant can be themed to any 1 color icon. The icon will not change colors in the various states, though it will change colors when disabled. https://github.com/cerner/terra-core/commit/5d1a6de0b4dd80fbbb620354ee5b823764386a5b